### PR TITLE
Prepare for JOSS submission

### PR DESCRIPTION
**Pull request by yangkcatiu**

### CODE REVIEW

File: birankpy/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+## pagerank
+```python
+pagerank(adj, d=0.85, max_iter=200, tol=0.0001, verbose=False)
+```
+
+Return the PageRank of the nodes in a graph.
+This funcion is replica of networkx's pagerank_scipy method with modification.
+See the original implementation in [networkx](https://networkx.github.io/documentation/networkx-1.10/_modules/networkx/algorithms/link_analysis/pagerank_alg.html#pagerank_scipy).
+
+This funciton takes the sparse matrix as input directly, avoiding the overheads
```
Comment (yy) [inline birankpy/README.md]: typo

File: birankpy/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# API documentation of birankpy
+
+## pagerank
+```python
+pagerank(adj, d=0.85, max_iter=200, tol=0.0001, verbose=False)
+```
+
+Return the PageRank of the nodes in a graph.
+This funcion is replica of networkx's pagerank_scipy method with modification.
```
Comment (yy) [inline birankpy/README.md]: what's the license of networkx?

Comment (yangkcatiu) [reply birankpy/README.md]: Good point. Let me try something else

Comment (yy) [reply birankpy/README.md]: they may allow modification/redistribution of the source code

**Comment by BrianAronson** [comment]
Should I go ahead and merge this?

**Pull request merged**